src/services/bcrypt.service.js

```diff
@@ -2,20 +2,29 @@ import bcrypt from 'bcrypt';
 
 /**
  * @exports
- * @class passwordHandler
+ * @class BcryptService
  */
 class BcryptService {
   /**
  * users can create an account
  * @static
  * @param {string} password
  * @param {string} Salt
- * @memberof passwordHandler
+ * @memberof BcryptService
  * @returns {object} hash
  */
   static hashPassword(password) {
-    return bcrypt.hashSync(password, bcrypt.genSaltSync(Number(process.env.SaltRounds)));
+    return bcrypt.hashSync(password, bcrypt.genSaltSync(Number(process.env.SALT_ROUNDS)));
   }
-}
 
+  /**
+   *
+   * @param {myPassword} myPassword
+   * @param {dbPassword} dbPassword
```

Comment:
@victor-abz  Can we use passwordFromDb instead of dbPassword  to avoid confusion with database password

Reply:
ok